config/zones/FR.yaml

```diff
@@ -290,3 +290,16 @@ sources:
   : link: https://unece.org/sites/default/files/2022-04/LCA_3_FINAL%20March%202022.pdf#page=37,
       https://proceedings.windeurope.org/biplatform/rails/active_storage/disk/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDRG9JYTJWNVNTSWhORFJ0ZDJJMWVUbG9OMll6TVRaaGEza3lkamgxZG1aM056WnZZZ1k2QmtWVU9oQmthWE53YjNOcGRHbHZia2tpQVk1cGJteHBibVU3SUdacGJHVnVZVzFsUFNKWGFXNWtaWFZ5YjNCbExWZHBibVF0Wlc1bGNtZDVMV2x1TFVWMWNtOXdaUzB5TURJeExYTjBZWFJwYzNScFkzTXVjR1JtSWpzZ1ptbHNaVzVoYldVcVBWVlVSaTA0SnlkWGFXNWtaWFZ5YjNCbExWZHBibVF0Wlc1bGNtZDVMV2x1TFVWMWNtOXdaUzB5TURJeExYTjBZWFJwYzNScFkzTXVjR1JtQmpzR1ZEb1JZMjl1ZEdWdWRGOTBlWEJsU1NJVVlYQndiR2xqWVhScGIyNHZjR1JtQmpzR1ZBPT0iLCJleHAiOiIyMDIyLTExLTAyVDE1OjU0OjAzLjEyNFoiLCJwdXIiOiJibG9iX2tleSJ9fQ==--c25280a7345257bd91bfaf6d64ddb75c55eef799/Windeurope-Wind-energy-in-Europe-2021-statistics.pdf?content_type=application%2Fpdf&disposition=inline%3B+filename%3D%22Windeurope-Wind-energy-in-Europe-2021-statistics.pdf%22%3B+filename%2A%3DUTF-8%27%27Windeurope-Wind-energy-in-Europe-2021-statistics.pdf
 timezone: Europe/Paris
+subZoneNames:
```

Comment:
Just wondering, how will the flowtracing pipeline react?
It will see data from France from both directly measured and also compute the aggregation data. So I guess we will have two datapoints for the same timestamp. One will probably replace the other as we are indexing by datetime. Since the aggregation is computed last, it will probably be the aggregation that will be in the database.

Reply:
Could you check how it works for Sweden? It should both have working parsers and Aggregates the subzones.

Note: for Sweden the source is exactly the same as we sum the bidding zones in the ENTSO-E parser.

Reply:
Ok thanks! I didn't know we were producing the SE data like this.
So the way I see it, the last step of the flowtracing pipeline computes spatial aggregations, which I think would be inserted after the flowtracing of parsed data. Therefore if a flowtraced data from parsed data exists with the same timestamp as the flowtraced data coming from the spatial aggregation step, it will be overwritten.

Reply:
Our aggregations should give the same as the national level (especially in the case of Sweden since we are doing it at parser level). But there might be differences we would need to explain.

Reply:
> So as it stands I think having a main zone defined like this would most likely overwrite the data obtained through the parsers and have it replaced with data obtained through the aggregation step.

We could remove the parser level aggregation for Sweden and see what happens? ENTSOE-E has country level data as well but it was missing solar and gas but it would be nice to do this anyway as it would save us a few API calls.

Reply:
@unitrium yes it's correct that the country-wide data will be overwritten by the aggregation if the subzones pass the quality criterion (for now we just check if all subzones have been flowtraced). We could change the quality check to make sure the country load is equal to the aggregated load.